SampleApp/app/src/main/java/com/pubmatic/openwrap/ottsampleapplication/MainActivity.java

```diff
@@ -85,10 +92,12 @@ protected void onCreate(Bundle savedInstanceState) {
         owAdsLoader.setAdsLoaderListener(new POWAdLoading.AdsLoaderListener() {
             @Override
             public void onAdReceived(@NonNull POWAdResponse response) {
-                JSONObject targetingJson = response.getTargeting();
-                String updatedGamAdsUrl = gamAdsUrl;
+                JSONArray targetingJson = response.getTargeting();
+                String cust_params = "dp=0&tool=video&artid=42733721&pos=preroll&";
```

Comment:
Formating for custom params can be done using the String formater (URL constant)or generic string builder.